### PR TITLE
feat(openapi): adapt OpenAPIv2 "Add data"

### PR DESCRIPTION
**Pull request by wangyuqing0424**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-27k14ph OpenAPIv2 对接 —— AddData](https://app.clickup.com/t/27k14ph)

File: graviti/openapi/data.py [hunk 1]
```diff
@@ -251,3 +251,71 @@ def update_data(
...
+        ...                     },
+        ...                 }
+        ...            ],
+        ... )
+
+    """
+    url = urljoin(url, f"v2/datasets/{owner}/{dataset}/drafts/{draft_number}/sheets/{sheet}/data")
+    post_data: Dict[str, Any] = {"data": data}
+    if strategy_params is not None:
+        post_data["arguments"] = strategy_params
```
Comment (graczhual) [inline graviti/openapi/data.py:319]: arguments -> strategy_params

**Review by graczhual — CHANGES_REQUESTED** [review]

File: graviti/openapi/data.py [hunk 2]
```diff
@@ -251,3 +251,71 @@ def update_data(
...
+    Examples:
+        >>> update_dataset(
+        ...     "ACCESSKEY-********",
+        ...     "https://api.graviti.com/",
+        ...     "czhual",
+        ...     "OxfordIIITPet",
+        ...     draft_number = 1,
+        ...     sheet = "train",
+        ...     data = [
+        ...                 {
```
Comment (linjiX) [inline graviti/openapi/data.py:290]: the indentation seems wrong

File: graviti/openapi/data.py [hunk 3]
```diff
@@ -251,3 +251,71 @@ def update_data(
...
+
+    Arguments:
+        access_key: User's access key.
+        url: The URL of the graviti website.
+        owner: The owner of the dataset.
+        dataset: Name of the dataset, unique for a user.
+        draft_number: The draft number.
+        sheet: The sheet name.
+        data: The update data.
+        strategy_argument: The dict about parameters of the Primary
```
Comment (graczhual) [inline graviti/openapi/data.py:278]: parameters  -> the argument

**Review by graczhual — CHANGES_REQUESTED** [review]

**Review by graczhual — APPROVED** [review]

**Pull request merged**